### PR TITLE
Support for remote environments + seasons cubemaps

### PR DESCRIPTION
**Pull request by keianhzo**
Support for remote environments + seasons cubemaps

### CODE REVIEW

File: versions.gradle [hunk 1]
```diff
@@ -32,7 +32,7 @@ versions.android_components = "44.0.0"
 // that we depend on directly for its rustlog package, and it's important
 // that it be kept insync with the version used by android-components above.
 versions.mozilla_appservices = "0.58.2"
-versions.mozilla_speech = "2.0.1"
+versions.mozilla_speech = "2.0.4"
```
Comment (bluemarvin) [inline versions.gradle:35]: This should probably be a different PR?

**Comment by keianhzo** [comment]
Moved all the envs but void and offworld to remote.
